README.md

```diff
@@ -25,7 +25,7 @@ n_adapts = 2_000
 θ_init = randn(D)
 
 # Define metric space, Hamiltonian and sampling method
-metric = DenseEuclideanMetric(θ_init)
+metric = UnitEuclideanMetric(D)
```

Comment:
Why do we change the example of using dense?

Reply:
If this is because of we don't have easy interface, see my comments below.

Reply:
It was, will revert with proposed interface below.